src/main/java/duke/Event.java

```diff
@@ -0,0 +1,76 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * Represents an Event that is accompanied by a specific date
+ * and time of occurrence.
+ */
+public class Event extends Task {
```

Comment:
May I suggest renaming using EventTask instead? I feel that an EventTask will more accurately reflect its inheritance from Task.